update_dependency_licenses.py

```diff
@@ -54,27 +70,53 @@ def GetPackageLicenses(package_name):
         else:
             match = re.search(r"^License:\s*(.*)$", metadata,
                               re.MULTILINE | re.IGNORECASE)
+            assert match
             package_license = re.sub(r"\sLicences|\sLicense,?|Version\s", "",
                                      match.group(1))
             package_license = re.sub("-", " ", package_license)
         match = re.search(r"^Home-page:\s*(.*)$", metadata,
                           re.MULTILINE | re.IGNORECASE)
+        assert match
         homepage = match.group(1)
         return package_license, homepage
-    return None
+    return None, None
+
+
+@dataclass(frozen=True)
+class PackageInfo:
+    """Represents the package metadata we are extracting."""
+    package: str
+    licence: str
+    category: str
+    homepage: str
+
+    def __lt__(self, other: "PackageInfo") -> bool:
+        return self.licence < other.licence
+
+    def __iter__(self):
+        return iter((self.package, self.licence, self.category, self.homepage))
+
+    @classmethod
+    def get_headers(cls) -> Tuple[str, ...]:
```

Comment:
Nit `Tuple[str, str, str, str]` tells me how many strings to expect.

Reply:
I actually did that on purpose.
As the comment suggests, `get_headers()` and `__iter__()` must be kept in sync and the code will self adjust to how many things we return (compared to having headers hard-coded in the caller which requires a stricter type checking)

Reply:
Ack